package.json

```diff
@@ -12,19 +12,19 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "npm run policy-check && npm run layer-check && npm run build:genver && npm run build:compile && npm run lint && npm run build:docs",
-		"build:ci": "npm run build:genver && lerna run build:compile --stream",
+		"build:ci": "lerna run build:compile --stream",
```

Comment:
It shouldn't stay here so that build the whole repo still generates the version file for the packages that care about it?

Reply:
You're absolutely right! I reverted all the release group root changes.